frontend/common/src/components/UserProfile/UserProfile.tsx

```diff
@@ -93,7 +93,7 @@ const UserProfile: React.FC<UserProfileProps> = ({ applicant, sections }) => {
         {showSection("about") && (
           <TableOfContents.AnchorLink id="about-section">
             {intl.formatMessage({
-              defaultMessage: "About",
+              defaultMessage: "About Me",
```

Comment:
The [issue](https://github.com/GCTC-NTGC/gc-digital-talent/issues/3390#issue-1309591901) has **About me** in sentence case but you are using title case. Which one did you want to use @gobyrne, **About me** or **About Me**?

Reply:
I think it may be odd to use sentence case since all of the other items are using title case.

<img width="211" alt="Screen Shot 2022-07-26 at 11 42 26 AM" src="https://user-images.githubusercontent.com/4127998/181050256-faea9cec-cff4-4df2-93fb-297bdebf1734.png">

Reply:
@gobyrne says use **About Me**.
i've added an issue related to this to make the case consistent (there are more items that use sentence case that are not included in the screenshot):

- https://github.com/GCTC-NTGC/gc-digital-talent/issues/3497